dev/io.openliberty.org.jboss.resteasy.cdi.ee11/src/org/jboss/resteasy/cdi/ResteasyCdiExtension.java

```diff
@@ -0,0 +1,349 @@
+package org.jboss.resteasy.cdi;
```

Comment:
I realize that the original file from jboss may not have had a copyright header but we should add a suitable one. One which matches their license etc. We should then add some more comments to say where this file was copied from and why we are changing it.

Reply:
e.g. copy the copyright header from https://github.com/resteasy/resteasy/blob/main/resteasy-cdi/src/main/java/org/jboss/resteasy/cdi/ContextProducers.java which is in the same package.

Reply:
This is something that affects all of our overridden files in a number of projects in EE11 and all previous releases.   I think it would be best to handle this under a separate issue that can be prioritized and tracked.    I'll create one and put a link here.

Reply:
This will be resolved under #26373